blockstore.go

```diff
@@ -504,6 +504,42 @@ Retry:
 	return err
 }
 
+func (b *Blockstore) DeleteMany(cids []cid.Cid) error {
```

Comment:
Could probably make it a variaric argument.

Reply:
Then i wouldnt interfacematch with vyzo: https://github.com/filecoin-project/lotus/blob/master/blockstore/badger/blockstore.go#L355

Reply:
Ok, right, and go is dumb.